src/hash_table.py

```diff
@@ -0,0 +1,28 @@
+# _*_ encoding: utf-8 _*_
+
+
+class HashTable(object):
+    """Create an instance of a hash table."""
+
+    def __init__(self, length):
+        """Initialize with a certain size of table."""
+        self.buckets = [[] for x in range(length)]
```

Comment:
Do you want the buckets themselves to be accessible? Or do you want them to be "private"?